README.md

```diff
@@ -28,7 +28,14 @@ packages are included:
 
 # Development
 
-The CLI uses lerna to manage multiple packages in a monorepo.
+The CLI uses lerna to manage multiple packages in a monorepo. Dependencies are hoisted during bootstrap by default.
+To disable this, modify the following in `lerna.json`.
+``` json
+"command": {
+    "bootstrap": {
+        "hoist": true
```

Comment:
Did you mean `"hoist": false` here?

Reply:
I just pasted what section should be edited instead of what to edit it to. I'll swap the boolean to be more clear.